restli-client/src/main/java/com/linkedin/restli/client/ParSeqBasedCompletionStageFactory.java

```diff
@@ -60,15 +60,39 @@ private void checkEngine()
    * build {@link ParSeqBasedCompletionStage} by using a {@link Task}
    * Note the input Task needs to run by the {@link Engine} so it can produce an output to create this Stage
    *
-   * @param task the input {@link Task} to create the stage, which needs to be run by the engine
-   * @return @link ParSeqBasedCompletionStage} instance
+   * @param task the input {@link Task} to create the stage, which needs to be run by the engine already
+   * @return {@link ParSeqBasedCompletionStage} instance
    */
   public ParSeqBasedCompletionStage<T> buildStageFromTask(Task<T> task)
   {
     checkEngine();
     return new ParSeqBasedCompletionStage<T>(_engine, _asyncExecutor,task);
   }
 
+  /**
+   * Convenient method to create a {@link ParSeqBasedCompletionStage} with a {@link Task} has not started.
+   *
+   * Will build {@link ParSeqBasedCompletionStage} by using a {@link Task} that has not started running, which will
+   *    be run by the engine provided in this factory.
+   *
+   *
+   *  Note0: In {@link ParSeqBasedCompletionStage}, all {@link Task} used was assumed to have started running and will
+   *         produce a value in the future.
+   *  Note1: Use {@link ParSeqBasedCompletionStage##buildStageFromTask(Task)} if the Task has started
+   *  Note2: Difference between this method and {@link ParSeqBasedCompletionStage##buildStageFromCompletionStage(CompletionStage)}
+   *         :If {@link Task} produces an exception, this method will have the very exception, while {@link ##buildStageFromCompletionStage(CompletionStage)}
```

Comment:
have the same exception

Reply:
change. But "very" is also an adjective...